src/utils/pokemon.d.ts

```diff
@@ -60,6 +60,7 @@ export namespace Pokemon {
     species: string;
     types: string[];
     weightkg: number;
+    evYields?: Stats<0 | 1 | 2 | 3>;
```

Comment:
```suggestion
    evYields: Stats<0 | 1 | 2 | 3>;
```

Eventually once all entries have been populated this needs to be changed

Reply:
Not all Pokemon have EV yields listed (mainly alternate formes), so this one I'm not entirely sure on but still (unless I've missed something and they share EV yields across formes?)

Reply:
Alternate formms generally have the same EV yields as the base form, unless otherwise specified.